pepper-apis/src/main/java/org/broadinstitute/ddp/json/FileUploadUrlResponse.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.ddp.json;
+
+import com.google.gson.annotations.SerializedName;
+
+public class FileUploadUrlResponse {
+    @SerializedName("url")
+    String url;
+
+    @SerializedName("guid")
+    String guid;
+
```

Comment:
As indicated in comment elsewhere, we should indicate the upload method (PUT or POST)

Reply:
Done